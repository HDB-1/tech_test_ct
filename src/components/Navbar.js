import React from 'react'
import {Link} from 'react-router-dom'
function NavBar() {
    return (
        <div>
            <Link to="/" >HOME</Link>
            <Link to="/info">Information</Link>
        </div>
    )
}

export default NavBar;