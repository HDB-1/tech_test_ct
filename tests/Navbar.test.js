import React from 'react'
import { shallow } from 'enzyme'
import Navbar from '../src/components/Navbar'


describe("shallow app", () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(<Navbar />)
    })
    it("should render correctly", () => {
        expect(wrapper).toMatchSnapshot();
    })
});

