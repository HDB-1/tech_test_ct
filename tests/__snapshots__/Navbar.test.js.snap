// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shallow app should render correctly 1`] = `
<div>
  <Link
    to="/"
  >
    HOME
  </Link>
  <Link
    to="/info"
  >
    Information
  </Link>
</div>
`;
